rnd/market/README.md

```diff
@@ -1,14 +1,37 @@
 # AutoGPT Agent Marketplace
 
 ## Overview
+
 AutoGPT Agent Marketplace is an open-source platform for autonomous AI agents. This project aims to create a user-friendly, accessible marketplace where users can discover, utilize, and contribute to a diverse ecosystem of AI solutions.
 
 ## Vision
+
 Our vision is to empower users with customizable and free AI agents, fostering an open-source community that drives innovation in AI automation across various industries.
 
-# Key Features
+## Key Features
 
 - Agent Discovery and Search
 - Agent Listings with Detailed Information
 - User Profiles
-- Data Protection and Compliance
\ No newline at end of file
+- Data Protection and Compliance
+
+## Getting Started
+
+To get started with the AutoGPT Agent Marketplace, follow these steps:
+
+- Copy `.env.example` to `.env` and fill in the required environment variables
+- Run `prisma migrate deploy`
+- Run `prisma generate`
+- Run `fastapi dev market/app.py`
```

Comment:
can we run with poetry? just thinking we should have a single way of doing all python apps for user consistency

Reply:
Actually I see below you already hve the poetry command - why the fastapi command too then?

Reply:
I'll tidy up the readme, thanks